src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        class="card__image"
+        alt="APPLE-A1419-iMac-27-picture"
+      >
+
+      <h3 class="card__item-name">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__product-code">
+      Product code: 195434
```

Comment:
same here, check indents for the content
```suggestion
        Product code: 195434
```